code/modules/mob/mob_helpers.dm

```diff
@@ -314,28 +314,7 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 	return sanitize(t)
 
 
-/proc/shake_camera(mob/M, duration, strength=1)
```

Comment:
Again, a comment showing to where this has moved might be useful

Reply:
DS13 no longer inherits from bay, its not really intended to be used as a base by other repos

This proc was silly to exist in mob helpers to begin with, since
1. Its not a mob proc, its global
2. It works on the client, not the mob

Reply:
This is true, but i'm more talking about tracking changes internally and it's good practice anyway.